src/parser/priest/shadow/modules/shadowlands/conduits/DissonantEchoes.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import EventHistory from 'parser/shared/modules/EventHistory';
+import { Options } from 'parser/core/Module';
+import { t, Trans } from '@lingui/macro';
+
+import Statistic from 'interface/statistics/Statistic';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import ConduitSpellText from 'interface/statistics/components/ConduitSpellText';
+
+import { DISSONANT_ECHOES_DAMAGE_INCREASE } from 'parser/priest/shadow/constants';
+
+class DissonantEchoes extends Analyzer {
+  static dependencies = {
+    eventHistory: EventHistory,
+  };
+  protected eventHistory!: EventHistory;
+
+  conduitRank = 0;
+  damage = 0;
+  procsGained = 0;
+  procsUsed = 0;
+
+  get procsWasted() {
+    return this.procsGained - this.procsUsed;
+  }
+
+  get procsWastedPercentage() {
+    return this.procsWasted / this.procsGained;
+  }
+
+  constructor(options: Options) {
+    super(options);
+
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.DISSONANT_ECHOES.id);
+    if (!this.conduitRank) {
+      this.active = false;
+      return;
+    }
+
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.VOID_BOLT), this.onDamage);
+    this.addEventListener(Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.DISSONANT_ECHOES_BUFF), this.onBuffApplied);
+    this.addEventListener(Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.DISSONANT_ECHOES_BUFF), this.onBuffRemoved);
+  }
+
+  onDamage(event: DamageEvent) {
+    const raw = event.amount + (event.absorbed || 0);
+    this.damage += raw - (raw / DISSONANT_ECHOES_DAMAGE_INCREASE);
```

Comment:
Its minor and you dont need to change it. But just incase you werent aware of its existance, you can do this via calculateEffectiveDamage

`this.damage += calculateEffectiveDamage(event, DAMAGE_BONUS)`

Would need to change your constant to .35 instead of 1.35 to do that of course. Again, dont need to change it, just pointing it out incase you didnt know it existed.